<?xml version="1.0"?>

<launch>
    <node pkg="holo_fingers" type="holo_fingers_node" name="holo_fingers" output="screen">
        <param name="input_topic" value="/pacman_vision/processed_scene"/>
        <param name="pass" value="0.015" />
    </node>

    <!-- launch the rgbd sensor, in this case using openni2 for the asus -->
    <include file="$(find openni2_launch)/launch/openni2.launch">
        <arg name="depth_registration" value="true" />
        <arg name="color_depth_synchronization" value="true" />
        <arg name="rgb_camera_info_url" value="file://$(find calibration)/config/intrinsics/rgb_asus1.yaml"/>
        <arg name="depth_camera_info_url" value="file://$(find calibration)/config/intrinsics/depth_asus1.yaml"/>
    </include>

    <include file="$(find pacman_vision)/launch/pacman_vision.launch">
        <arg name="run_rviz" value="true"/>
        <arg name="load" value="holo"/>
    </include>
</launch>
